### PR TITLE
adds support for watch count threshold

### PR DESCRIPTION
**Pull request by daniellmb**
also adds warning color at 70% of metric threshold

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -113,7 +114,7 @@ function showAngularStats(opts) {
       background: 'black',
       borderBottom: '1px solid #666',
       borderRight: '1px solid #666',
-      color: 'red',
+      color: '#666',
```
Comment (kentcdodds) [inline src/index.js]: This color is very different from the original `red` value here. Perhaps I'm missing something about the logic?

**Comment by kentcdodds** [comment]
Also, it doesn't appear that this has anything that would cover the console logs. Were you going to update those as well?

File: src/index.js (hunk 1 above)
Comment (daniellmb) [inline src/index.js]: the colors for the watch count and digest count are set separately now (see around lines: [221](https://github.com/daniellmb/ng-stats/commit/f1fd7ae333e2aefda094ce99d9bd1a31c85b8b71#diff-1fdf421c05c1140f6d71444ea2b27638R221) and [230](https://github.com/daniellmb/ng-stats/commit/f1fd7ae333e2aefda094ce99d9bd1a31c85b8b71#diff-1fdf421c05c1140f6d71444ea2b27638R230)). The color in this section is just for the " | " pipe delimiter between the values.

**Comment by daniellmb** [comment]
yep, both the console and canvas functions got updated and pull the color from a common util function.

File: src/index.js (hunk 1 above)
Comment (kentcdodds) [inline src/index.js]: Ah, I see! I'll pull this and check it out locally. Thanks!

**Comment by kentcdodds** [comment]
Wow! This is super awesome! Thanks for working on it! I'll merge and release in a second.

**Comment by daniellmb** [comment]
sweet :)

**Pull request merged**